lib/label.js

```diff
@@ -7,35 +7,45 @@ const request = require('request-promise');
 const assert = require('assert');
 
 class Label {
+
   constructor(owner, repo, token) {
     this.owner = owner;
     this.repo = repo;
     this.token = token;
-    this.rp = request.defaults({ 
+    this.rp = request.defaults({
       baseUrl: `${GH_HOST}/repos/${this.owner}/${this.repo}`,
       headers: {
+        // Opt-in into the preview version in order to be able to work with label descriptions
+        // See <https://github.com/octokit/octokit.rb/issues/1001#issuecomment-379505671>
+        Accept: 'application/vnd.github.symmetra-preview+json',
         'user-agent': 'gh-label',
       },
       json: true
     });
- 
   }
 
   getAll() {
     const options = {
-      uri: `/labels`,
+      uri: '/labels',
     };
     if (this.token) {
       options.headers = {
-        Authorization: `token ${this.token}` //needed if private repo
+        Authorization: `token ${this.token}` // Needed if private repo
       };
     }
 
     return this.rp(options)
       .then(res => {
         debug('original lables: %j', res);
         return res.map(label => {
-          return { name: label.name, color: label.color };
+          const simplifiedLabel = {
+            color: label.color,
+            name: label.name,
+          };
+          if (label.description) { // Only use actual description values (not "null")
```

Comment:
FYI: I know this can be written in a more elegant way using optional object destructuring, but I didn't want to break compatibility with older NodeJS versions as the README mentions NodeJS 4.4 and above as the prerequisite.

Reply:
sure, not a big problem here.